### ISSUE TITLE
Не работает разметка страницы Hello World

### ISSUE THREAD

**Issue by DBMainUser** [EvilBeaver/webapp-template#1]
Шел по шагам статьи https://infostart.ru/public/783435/
После шага "И что получилось?" у вас картинка:
![default](https://user-images.githubusercontent.com/35705084/36169442-11b179dc-111e-11e8-9870-c4790ee17b38.png)
У меня картинка:
![default](https://user-images.githubusercontent.com/35705084/36169390-f50221b0-111d-11e8-901e-7e194dfc940c.png)
Консоль браузера пишет что страница не содержит css-сетки. Может в этом дело? Как её включить? Ведь файлик Default.css уже создан в проекте.
Консоль браузера:
![default](https://user-images.githubusercontent.com/35705084/36170467-44fe30e8-1121-11e8-966d-1d8e70e202dc.png)

**Comment by nixel2007** [comment]
Регистр файлов совпадает?

**Comment by EvilBeaver** [comment]
Да, очень похоже, что default.css не нашелся. Он должен лежать в папке wwwroot

**Comment by DBMainUser** [comment]
Но физически то файл создан и лежит на своем месте. Это баг или фича)?
Пробовал менять строку link rel="stylesheet" href="~/default.css"/ на 
link rel="stylesheet" href="C:\ServersConsole\webapp-template-master\src\wwwroot\default.css"
Картинка при этом меняется на: 
![default](https://user-images.githubusercontent.com/35705084/36186628-0a8959c4-1162-11e8-85a4-7bc8bfdfb23f.png)
То есть какие то настройки берутся из default.css, но как то некорректно.

**Comment by EvilBeaver** [comment]
Кажется я понял. Инфостарт искажает пример кода со страницы. У вас отсутствует не default.css а semantic ui, потому что инфостарт заменил его на свой редирект. Возьмите примеры кода отсюда: https://github.com/EvilBeaver/osp-articles/tree/master/chapter-01
